tests/indent/ecma/func.js

```diff
@@ -49,3 +49,11 @@ function prepare_list_fetcher(filter) {
     { key: ['/list', filter] }
   )
 }
+
+console.log({
+  key: 'value',
+}, {
+  key: 'value',
+},
+  hi
+);
```

Comment:
Umm
Pretty sure this will fail, and it's pretty common as well
```
console.log(
  {
    key: 'value',
  },
  {
    key: 'value',
  },
  hi
);
```

What we need to check here is whether the start of the object/array/function node is on the same line of the place where ( is placed

Reply:
Ah, damn. You're right, looks like this one needs more thought